web_monitoring/differs.py

```diff
@@ -51,11 +52,20 @@ def pagefreezer(a_url, b_url):
     # Just send PF the urls, not the whole body.
     # It is still useful that we downloaded the body because we are able to
     # validate it against the expected hash.
-    return web_monitoring.pagefreezer.compare(a_url, b_url)
+    obj = web_monitoring.pagefreezer.PageFreezer(a_url, b_url)
```

Comment:
Yeah, you're right. The `__init__` operations could lead to extremely slow responses when we're handling a large number of diffs. 
This reminds me of the question I had about our go-to diff service.
Given the uncertainty around the external services, this diff could probably be non-existent in the future. Do we have any updates on that yet?